app/views/content_files/_show.html.erb

```diff
@@ -0,0 +1,22 @@
+<tr>
```

Comment:
Trivial, but: should we stick to two-space indents in ERBs? I fear our editors are going to keep introducing large whitespace diffs, but maybe that's no big deal.

Reply:
Fixed.